src/views/GlossaryPage.vue

```diff
@@ -0,0 +1,279 @@
+<template>
+  <DashboardLayout>
+    <DashboardCard>
+      <template #title> Agencies and Organizations </template>
+      <template #content>
+        <div class="px-4" style="font-size: 0.95rem">
+          <p v-for="agency in agencyAcronyms" :key="agency.acronym">
+            <strong style="font-size: 1.02rem">{{ agency.acronym }}:</strong>
+            {{ agency.name }}
+          </p>
+        </div>
+      </template>
+    </DashboardCard>
+    <DashboardCard>
+      <template #title>Models and Data</template>
+      <template #content>
+        <div class="px-4" style="font-size: 0.95rem">
+          <p v-for="model in modelAcronyms" :key="model.acronym">
+            <strong style="font-size: 1.02rem">{{ model.acronym }}:</strong>
+            {{ model.name }}
+          </p>
+        </div>
```

Comment:
this chunk of markup seems to be repeated across all three cards - consider pulling it out into it's own little component (`GlossaryDefinition` perhaps)

Reply:
This one will take some more learning to figure out the right way to do it. I started down the rabbit hole and couldn't quite figure it out. For now I made a `GlossaryDefinitions` component, but couldn't figure out how to use it.